File: winery-frontend/src/types/podrum.ts
export interface Podrum {
  idpod: number
  temp: number
  nazivpod: string
  brojBuradi: number
}

export interface PodrumDetail {
  idpod: number
  temp: number
  nazivpod: string
  brojBuradi: number
}

export interface CreatePodrumRequest {
  temp: number
  nazivpod: string
}

export interface UpdatePodrumRequest {
  temp: number
  nazivpod: string
}

File: WineryAPI/Models/Podrum.cs
using System;
using System.Collections.Generic;

namespace WineryAPI.Models;

public partial class Podrum
{
    public int Idpod { get; set; }

    public decimal Temp { get; set; }

    public string Nazivpod { get; set; } = null!;

    public virtual ICollection<Bure> Bures { get; set; } = new List<Bure>();
}
